smart_control/environment/environment.py

```diff
@@ -1217,12 +1217,6 @@ def action_spec(self) -> types.NestedArraySpec:
   def observation_spec(self) -> types.NestedArraySpec:
     return self._observation_spec
 
-  def _format_action(
-      self, action: types.NestedArray, action_names: Sequence[str]
```

Comment:
unused arg `action_names`. 

this function is a no-op. so we are removing the function as well, after confirming no child class in the Google codebase uses this (see PR notes)